### ISSUE TITLE
Missing keyboard modifications

### ISSUE THREAD

**Issue by luklewluk** [banhbaoxamlan/X230-Hackintosh#23]
Hi

I can't find `SSDT-X220-KBD.aml` mentioned in https://github.com/banhbaoxamlan/X230-Hackintosh/tree/master/Other/Modifications/Keyboard .

Where could I get it?

Thank you

**Comment by luklewluk** [comment]
Actually I don't think if dedicated `.aml` for X220 keyboard is needed because everthing seems to be working fine with the default `SSDT-KBD.aml` 🤔

**Comment by lese9855** [comment]
When I used SSDT-KBD.aml on the 7 row keyboard, I found that "the right command key" worked as "the left option key".
It seems to be due to the difference in keyboard layout between the 6 row keyboard and the 7 row keyboard.

So I edited SSDT-KBD.aml to fix this.
I uploaded the file here. Just replace the original one with this.

[SSDT-KBD.aml.zip](https://github.com/banhbaoxamlan/X230-Hackintosh/files/5541035/SSDT-KBD.aml.zip)
